### PR TITLE
update script sources to be even with the example file ...

### PR DESCRIPTION
**Pull request by cristovaov**
... in reactjs/react-tutorial repo

cf. https://github.com/reactjs/react-tutorial/issues/57

### CODE REVIEW

File: docs/docs/tutorial.md [hunk 1]
```diff
@@ -39,9 +39,9 @@ For this tutorial, we'll use prebuilt JavaScript files on a CDN. Open up your fa
 <html>
   <head>
     <title>Hello React</title>
-    <script src="https://fb.me/react-{{site.react_version}}.js"></script>
-    <script src="https://fb.me/JSXTransformer-{{site.react_version}}.js"></script>
-    <script src="https://code.jquery.com/jquery-2.1.3.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/react/0.13.0/react.js"></script>
```
Comment (zpao) [inline docs/docs/tutorial.md]: We should leave this with `{{site.react_version}}` in there. That ensures we don't need to update this page for every release.

Comment (cristovaov) [inline docs/docs/tutorial.md]: @zpao should be fixed with 4564e555b3b815f1ac8fcf9fd46eb61412b556aa ~

**Comment by zpao** [comment]
Great. 1 more request - could you flatten these to a single commit?

**Comment by cristovaov** [comment]
ping @zpao -- don't know if you had been notified that commits have been flatten as requested.

**Comment by zpao** [comment]
Ah, nope. Github doesn't send notifications for commit changes. Looks great, thanks!

**Comment by cristovaov** [comment]
Cheers! You tested my git-via-sourcetree knowledge there, to squash commits to one ;)

**Pull request merged**